### PR TITLE
fix 'Discontiguous selection is not supported' on chrome

### LINKED ISSUE #3: Getting 'Discontiguous selection is not supported' when making multiple copies on chrome 57
It is being fixed when moving the `window.getSelection().removeAllRanges()` before the `var range = new Range()`

### PR DESCRIPTION
**Pull request by borisirota**
fixes #3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +18,6 @@ function clipboardCopy (text) {
     successful = document.execCommand('copy')
   } catch (err) {}
 
-  window.getSelection().removeAllRanges()
```
Comment (feross) [inline index.js]: Please don't remove this line. Keep it in both places.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "clipboard-copy",
   "description": "Lightweight copy to clipboard for the web",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (feross) [inline package.json]: Please don't change the package version in a PR. That is the job of the maintainer to do when they publish a new version. If someone else sends a PR that I merge and release first, then the version you set here will be wrong.

**Review by feross — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (borisirota) [reply index.js]: Done

File: package.json (hunk 2 above)
Comment (borisirota) [reply package.json]: Sure. Done.

**Comment by feross** [comment]
Thanks, this is fixed in 1.1.0

**Pull request closed without merging**